internal/mode/static/state/graph/graph_test.go

```diff
@@ -501,9 +554,57 @@ func TestIsReferenced(t *testing.T) {
 			graph:    graph,
 			expected: false,
 		},
+
+		// Service tests
+		{
+			name:     "Service in graph's ReferencedServicesNames passes",
```

Comment:
I recommend using passes/fails in test names, because in unit tests, pass/fail applies to a test case.  So a test case either passes or fails.

For example, we can replace "Service in graph's ReferencedServicesNames passes" with "Service is referenced"

Reply:
Did you mean to say you don't recommend using passes/fails in test names?

Reply:
> Did you mean to say you don't recommend using passes/fails in test names?

my bad. yep, I don't recommend using passes/fails in test names